chapter_12/4_ Persist_Homology.tex

```diff
@@ -0,0 +1,71 @@
+%Contributor: Erica Wei
```

Comment:
The file name has a space between 4_ and Persist which breaks compilation, please remove it.